src/frontend/src/components/FilterPanel/components/CollectionDateFilter/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { Button, Menu, MenuItem } from "czifui";
+import { useFormik } from "formik";
+import { noop } from "lodash";
+import React, { FC, useEffect, useState } from "react";
+import DateField from "src/components/DateField";
+import {
+  DATE_ERROR_MESSAGE,
+  DATE_REGEX,
+} from "src/components/DateField/constants";
+import * as yup from "yup";
```

Comment:
not sure how big this library is, but maybe would be good to only import what we need?

Reply:
A cool thing! You can check lib sizes on [bundlephobia](https://bundlephobia.com/package/yup@0.32.9). Also, afaict yup only has a default export, so importing only what we need isn't possible in the way you're suggesting (I think).